ideascube/blog/templates/blog/content_form.html

```diff
@@ -19,12 +19,13 @@
                     {% csrf_token %}
                     <h3>{% trans 'Metadatas' %}</h3>
                     {% for field in form %}
-                        {% if field.name != 'text'  %}
+                        {% if field.name != 'text' and field.name != 'summary' %}
                             {% form_field field %}
                         {% endif %}
                     {% endfor %}
                 </div>
                 <div class="col pure-u-1 pure-u-md-2-3 layout-element">
+                    {% form_field form.summary %}
```

Comment:
I would argue the `summary` field should be below the `text` one.

(and we should make the `summary` optional, but that's for another ticket)